roofit/roofitcore/inc/RooCmdArg.h

```diff
@@ -116,6 +122,12 @@ class RooCmdArg : public TNamed {
   RooArgSet* _c ;        // Payload RooArgSets 
   RooLinkedList _argList ; // Payload sub-arguments
   Bool_t _prefixSubArgs ; // Prefix subarguments with container name?
+
+  using DataCollection = std::vector<std::unique_ptr<TObject>>;
+  std::shared_ptr<DataCollection> _sharedData; //!
+
+  // the next RooCmdArg created will take ownership of this data
+  static DataCollection nextSharedData;
```

Comment:
I don't get this. Why not use `make_shared` when you need a new one?

Reply:
There was some naming convention for statics as well. You can check e.g. RooAbsArg.h

Reply:
Okay, I will update! In RooAbsArg, the static member names begin with an underscore, so I'll follow that.